lib/packages/core/src/api/composeApi.ts

```diff
@@ -59,6 +61,8 @@ export function compose(settings: ApiSettings) {
             getAccountTransactions: getAccountTransactions(service),
             getUnconfirmedAccountTransactions: getUnconfirmedAccountTransactions(service),
             getAccountBalance: getAccountBalance(service),
+            generateSendTransactionQRCode: generateSendTransactionQRCode(service),
+            generateSendTransactionQRCodeAddress: generateSendTransactionQRCodeAddress(service),
```

Comment:
Does it make sense to expose the generator?

Reply:
Yes 100% IMO

Reply:
you explained, and seems valid